ui/qubemanager.ui

```diff
@@ -867,10 +867,10 @@ DisposableVMs</string>
     <bool>true</bool>
    </property>
    <property name="text">
-    <string>Is Template for DisposableVMs</string>
+    <string>DisposableVM Template</string>
    </property>
    <property name="toolTip">
-    <string>Is Template for DisposableVMs</string>
+    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Disposable VM template&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Allows using this qube as a template for Disposable VMs. The Disposable VMs will inherit the VM's state (configuration, installed programs etc.), but their state will not be persistent. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
> but their state will not be persistent.

I wonder if it would be clearer to say that they will be permanently destroyed when the initial window is closed. I wonder if there's a more intuitive way to say "the initial window."

Any ideas, @marmarek?

Reply:
Yeahhh, let's not get into service calls when non-terminal users can see it :) Maybe "their state will not persist between restarts"?

Reply:
> Maybe "their state will not persist between restarts"?

Sounds good.